jax/_src/xla_bridge.py

```diff
@@ -247,6 +248,21 @@ def make_cpu_client() -> xla_client.Client:
 )
 
 
+def _check_cuda_compute_capability(devices_to_check):
+  from jaxlib.gpu_triton import get_compute_capability
```

Comment:
I wonder if we do need the try-except in case someone creates a GPU client with a CPU jaxlib. I guess we can add it later.

Reply:
It is indeed weird, but wouldn't openxla/xla#9670 help with moving it out of `gpu_triton`.

Reply:
I've updated the PR to now use the same mechanism as `gpu_triton`, resulting in a bit of code duplication.